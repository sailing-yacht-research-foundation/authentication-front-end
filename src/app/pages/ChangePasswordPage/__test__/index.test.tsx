
import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { ChangePasswordPage } from '..';

const shallowRenderer = createRenderer();

describe('<ChangePasswordPage />', () => {
  it('should render to match the snapshot', () => {
    shallowRenderer.render(<ChangePasswordPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});