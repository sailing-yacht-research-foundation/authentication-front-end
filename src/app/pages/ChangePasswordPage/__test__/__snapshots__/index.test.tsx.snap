// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangePasswordPage /> should render to match the snapshot 1`] = `
<Row
  align="top"
  justify="center"
  style={
    Object {
      "background": "#f7f7f9",
      "minHeight": "100vh",
    }
  }
>
  <ChangePasswordForm />
</Row>
`;
